index.js

```diff
@@ -1,9 +1,14 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
 */
-module.exports = function(content) {
-	this.cacheable && this.cacheable();
-	this.value = content;
-	return "module.exports = " + JSON.stringify(content);
+module.exports = function(source) {
+  this.cacheable && this.cacheable();
```

Comment:
We don't support webpack 1.x. Cachable can be removed at this point

Reply:
Ok